### DISCUSSION TITLE
Make Everything the main page and remove from the navbar.

### DISCUSSION THREAD

**Discussion by paulmortlock** [sam-ludlow/mame-ao#30] [Ideas]
I like Everything page and would like this to be the main page.

**Comment by sam-ludlow** [comment]
Hello Paul

The design was like this so MAME novices would start with a list of fully working arcade machines. That all just work on the first click.

But I know what you mean if you want to just search you want everything really.

I will this about this.

Could even make it configurable.

**Reply by paulmortlock** [reply]
Nice a little option to switch it would be cool.

**Reply by paulmortlock** [reply]
How about making the grid table a tad more compact style wise and be able to sort the grid as well. Isn't there a bootstrap grid table pager thingy. And the logo could have the whole thing showing. I was going to try and make the UI a bit nicer but will refrain from doing so until you respond.

**Comment by sam-ludlow** [comment]
Well if anyone wants to build an alternate UI fill your boots.

It could be hosted on a web connecting to the API on localhost.

I may build a PoC using React or something.

But I have quiet enjoyed building the pure JS UI in a single file.
